src/thunderbots/software/gui/drawing/navigator.cpp

```diff
@@ -0,0 +1,57 @@
+#include "software/gui/drawing/navigator.h"
+#include <QtWidgets/QGraphicsScene>
+#include "software/geom/segment.h"
+#include "software/gui/geom/geometry_conversion.h"
+
+AIDrawFunction drawNavigator(std::shared_ptr<Navigator> navigator) {
+    static unsigned int count = 0;
+    count++;
+    // Thought we were catching thenavigator between superclass and
+    // subclass initialization. Apparently not....? Or at least this
+    // isn't the entire problem
```

Comment:
Confused about what you mean here? Also `thenavigator` -> `the navigator`

Reply:
all removed